src/components/FeedbackWidget.tsx

```diff
@@ -197,22 +203,22 @@ const FeedbackWidget = ({ className }) => {
     trackCustomEvent({
       eventCategory: `FeedbackWidget toggled`,
       eventAction: `Clicked`,
-      label: `Closed feedback widget`,
+      eventName: `Closed feedback widget`,
     })
   }
   const handleOpen = () => {
     setIsOpen(true)
     trackCustomEvent({
       eventCategory: `FeedbackWidget toggled`,
       eventAction: `Clicked`,
-      label: `Opened feedback widget`,
+      eventName: `Opened feedback widget`,
     })
   }
   const handleSubmit = (choice) => {
```

Comment:
Lets define types for the function args + the return value
```suggestion
  const handleSubmit = (choice: boolean): void => {
```